tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/temporal/TableauEncoder.scala

```diff
@@ -33,6 +34,21 @@ class TableauEncoder(
 
   private def inBoolSet(element: TBuilderInstruction): TBuilderInstruction = builder.in(element, builder.booleanSet())
 
+  /* replaces all occurences of foo := bar with foo = bar */
```

Comment:
Could you add a comment here why this is necessary?

Reply:
Just saw that this is similar to Igor's comment.
I think it's also valuable to mention why this is done at all.

Reply:
There is a short explanation where this is invoked: https://github.com/informalsystems/apalache/blob/6d5a5280aa66b7f3584188ea810f40c6447411f7/tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/temporal/TableauEncoder.scala#L388

do you think this is sufficient?

Reply:
Oh yes, that makes sense. ty